open_humans/urls.py

```diff
@@ -110,6 +110,9 @@
     url(r'^public-data-api/$',
         views.PublicDataDocumentationView.as_view(),
         name='public-data-api'),
+    url(r'^grant-projects/$',
+        views.GrantProjectView.as_view(),
+        name='project-grants'),
```

Comment:
I think we'd best rename the name to `'grant-projects'` to match the template and view and URL, especially since there's already a template `project-grants.html`